resources/scripts/beats/metricbeat-logs.yml

```diff
@@ -0,0 +1,27 @@
+---
+metricbeat.modules:
+- module: docker
+  metricsets:
+    - "container"
+    - "cpu"
+    - "diskio"
+    - "event"
+    - "healthcheck"
+    - "info"
+    - "memory"
+    - "network"
```

Comment:
Use https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-docker.html#_example_configuration_17

Maybe we could try with less number of metricsets

Reply:
```
% ecslog  docker-16-release-metricbeat.log-20220927.ndjson -i metricset.name  | grep metricset | sort | uniq -c | sort -n
   9     metricset: {"name": "info"}
  39     metricset: {"name": "healthcheck"}
  63     metricset: {"name": "network"}
  72     metricset: {"name": "container"}
  72     metricset: {"name": "cpu"}
  72     metricset: {"name": "diskio"}
  72     metricset: {"name": "memory"}
 657     metricset: {"name": "event"}
```

If it helps, here are the numbers of each of those metricset types for this build: https://apm-ci.elastic.co/job/apm-agent-nodejs/job/apm-agent-nodejs-mbp/job/PR-2950/13/